beaker-abs.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "beaker", "~> 4.0"
+  spec.add_dependency "vmfloaty", "~> 0.11"
```

Comment:
we should based this on the next version, which does the vmpooler GET requests using the user's token

and you need to use a file like 
```
# file at ~/.vmfloaty.yml
services:
  abs:
    url: 'https://abs/api/v2'
    type: 'abs'
    user: 'samuel'
    token: 'foo'
  vmpooler:
    url: 'http://vmpooler'
    user: 'samuel'
    token: 'bar'
```